### DISCUSSION TITLE
milvus concurrency? How many simultaneous search queries/insertions can milvus handle?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#18179] [Q&A and General discussion] [answered]
milvus concurrency? How many simultaneous search queries/insertions can milvus handle?

**Comment by yhmo** [comment, marked as answer]
Concurrency performance is mainly depends on several aspects: the hardware, deployment, dataset size, index type/parameters, search parameters, etc. It is hard to give an estimation, most of users get conclusion from testing and practice.
